src/main/java/genelectrovise/magiksmostevile/core/MagiksMostEvile.java

```diff
@@ -0,0 +1,84 @@
+package genelectrovise.magiksmostevile.core;
+
+import java.util.stream.Collectors;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import genelectrovise.magiksmostevile.core.setup.RegistryCreationManager;
+import genelectrovise.magiksmostevile.particle.ParticleClientStartup;
+import genelectrovise.magiksmostevile.particle.ParticleCommonStartup;
+import genelectrovise.magiksmostevile.registry.orbital.OrbitalRegistryGenerator;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.eventbus.api.IEventBus;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.InterModComms;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
+import net.minecraftforge.fml.event.lifecycle.InterModEnqueueEvent;
+import net.minecraftforge.fml.event.lifecycle.InterModProcessEvent;
+import net.minecraftforge.fml.event.server.FMLServerStartingEvent;
+import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
+
+@Mod(MagiksMostEvile.MODID)
+@Mod.EventBusSubscriber(modid = MagiksMostEvile.MODID, bus = Mod.EventBusSubscriber.Bus.MOD)
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Hi @aimiebell 
I will close this pull request because it is not currently needed.
I have opened a new issue for this: https://github.com/GenElectrovise/MagiksMostEvile/issues/27
Let's continue the conversation on there :)